src/logic/tokens/api/token.dto.ts

```diff
@@ -0,0 +1,8 @@
+export interface TokenDTO {
```

Comment:
@mikheevm just curious, why is this called DTO? are we doing any data transformation?

Reply:
data transfer object

Reply:
https://en.wikipedia.org/wiki/Data_transfer_object